services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -73,31 +78,31 @@ class FlowHelper {
                 false, false, "autotest flow", null, null)
     }
 
+    /**
+     * Adds flow with checking rules on source and destination switches.
+     * It is supposed if rules are installed on source and destination switches, the flow is completely created.
+     */
+    FlowPayload addFlow(FlowPayload flow, boolean checkStatus = true, boolean checkRules = true) {
```

Comment:
I wonder if boolean parameters are really required.
If users don't want any checks, wouldn't they be better off with northbound.addFlow()?